components/Table/Table.vue

```diff
@@ -2,15 +2,26 @@
   <table>
     <thead>
       <tr>
-        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">{{header.text}}</th>
+        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">
+          <slot :name="`header_${header.key}`" :header="header"> 
+            {{header.text}} 
+            </slot>
+        </th>
       </tr>
     </thead>
     <tbody>
-      <tr v-for="(item, j) in items" :key="j">
-        <td v-for="(header,i) in headers" :key="i" :style="textAlign(header.align)">
-          <slot :name="header.key" :item="item">{{ item[header.value] }}</slot>
-        </td>
-      </tr>
+      <template v-for="item in items">
+        <tr :key="item.length" @click="expandDetail(item.id)">
+          <td v-for="(header,i) in headers" :key="i" :style="textAlign(header.align)">
+            <slot :name="`item_${header.key}`" :item="item">{{ item[header.value] }}</slot>
+          </td>
+        </tr>
+        <tr v-if="showExpand && toggle.includes(item.id)" :key="item.id">
```

Comment:
the key here is using `item.id` where before it was `item.lenght`. This key should be the same, that's the key for the `v-for`. So same comment here, use the index from the `v-for`

Reply:
Done.